src/selectField/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { Ref } from 'react'
+import cc from 'classcat'
+import { ChevronIcon } from 'icon'
+import { color } from '_utils/branding'
+import { CommonFieldsProps } from '_utils/interfaces'
+import style from 'selectField/style'
+
+interface SelectFieldProps extends CommonFieldsProps {
+  readonly ariaLabel?: string
+  readonly options: Object[]
```

Comment:
Can you add a specific object like we did for `autocompleteItem`?
https://github.com/blablacar/ui-library/blob/master/src/typings/autocomplete.d.ts

Reply:
Done!